### DISCUSSION TITLE
Shelly 1 with contact sensor on gpio5

### DISCUSSION THREAD

**Discussion by spiccinocchi** [RavenSystem/esp-homekit-devices#2290] [MEPLHAA Scripts]
Can anyone help me with MEPLHAA script for a Shelly 1 with contact sensor on gpio5?

Is it correct this script:
{"c":{"io":[[[4],2],[[5],6,0,1],[[3],6,1]]},"a":[{"0":{"r":[[4]]},"1":{"r":[[4,1]]},"es": [{"t": 5,"f0": [[ 5,0 ]],"f1": [[ 5 ]]}]}]}

Thanks

**Comment by RavenSystem** [comment]
All is right. But `[[3],6,1]` is useless, because you don't use GPIO 3. Remove it:
```json
{"c":{"io":[[[4],2],[[5],6,0,1]]},"a":[{"0":{"r":[[4]]},"1":{"r":[[4,1]]},"es": [{"t":5,"f0":[[5,0]],"f1":[[5]]}]}]}
```

**Reply by spiccinocchi** [reply]
Yes, you are right I forgot to remove it

Thank you very much
